cob_bringup/robots/raw3-5.xml

```diff
@@ -23,17 +23,35 @@
 		</include>
 
 		<!-- startup hardware -->
-		<include file="$(find cob_bringup)/drivers/laser_lms1xx.launch" >
+		<include file="$(find cob_bringup)/drivers/sick_lms1xx.launch" >
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="name" value="base_laser_front" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/laser_lms1xx.launch" >
+		<include file="$(find cob_bringup)/drivers/sick_lms1xx.launch" >
 			<arg name="robot" value="$(arg robot)" />
 			<arg name="name" value="base_laser_rear" />
 		</include>
-		<include file="$(find cob_bringup)/drivers/relayboard.launch" >
+		<!--<include file="$(find cob_bringup)/drivers/relayboard.launch" >
 			<arg name="sim" value="true" />
+		</include>-->
```

Comment:
Please remove relayboard sim
According to https://github.com/ipa320/cob_robots/issues/403#issuecomment-179844477, the functionality is taken over by phidgets